RELEASE.md

```diff
@@ -1,3 +1,40 @@
+# Release 2.2.0
+
+The 2.2 minor series tracks TensorFlow 2.2.
+
+## Features
+
+- Profile plugin now should be pip installed.
+- Debugger:
+  - Added support for alerts (#3285)
+  - Support FULL_HEALTH debug mode (#3343)
+  - Integration with ngrx storage (#3359)
+- Add S3_ENDPOINT variable (#3347)
+- Confirm that the connection to tensorboard works or change to localhost (#2371)
+- Update --reload_multifile_inactive_secs default to 24 hours (#3243)
+
+## TensorBoard.dev updates
+
+- Improvements to [TensorBoard.dev] support:
+  - New `tensorboard dev update-metadata` command allows for updating the name and
+    description of experiments (#3277)
+  - Handshake protocol allows for more graceful control of which plugins are supported
+    for upload (#3299)
+  - Improved organization of artifacts downloaded during export (#3307)
+  - Adds `--plugins` option to uploader (#3377)
+  - Support for exporting blob data (#3373)
+
+## Bug fixes
+
+- Fix for the tooltip not disappearing issue (#3347)
```

Comment:
Please clarify what “the tooltip not disappearing issue” is: something
like “#3282 - Tooltips now properly disappear when the mouse pointer is
slowly moved out of the chart body (PR #3347)”.

In general, we try to make the release notes self-contained and helpful
to readers, which often means providing a bit more context than is in
the pull request or issue title.

Note also the reference to both the bug and the fix PR; see previous
“Bug fixes” sections for examples.